dla_cnn/desi/model_v2.1.py

```diff
@@ -0,0 +1,243 @@
+""" TensorFlow Models for DLA finder"""
+
+"""
+0.  Convert the model to TF 2.1
+
+used the tf_upgrade_v2 --intree model_v1  --outtree model_v2 to update this code to TF2.1
+But tf.summary.scalar requires manual check. 
+The TF 1.x summary API cannot be automatically migrated to TF 2.0, so symbols have been converted to tf.compat.v1.summary.* 
+and must be migrated manually.
+"""
+
+
+def weight_variable(shape):
```

Comment:
Please add doc strings to all of these methods.
They weren't there previously, but it would be good
to have them now.  And it would confirm that you 
know what each one is doing.

Reply:
I have added the tensorflow official update guid link and the repot.txt file google doc link

Reply:
Sorry, what i mean by doc strings are notes like
this at the start of each method:

```
        read Desi Mock spectrum from a fits file, load all spectrum as a DesiMock object
        :param spec_path:  str, spectrum file path
        :param truth_path: str, truth file path
        :param zbest_path: str, zbest file path
        :return: self.wavelength,self.data(contained all information we need),self.split_point_br,self.split_point_rz,self.data_size
```

Reply:
Dear X:
I have uploaded a notebook about runing this model_v2,but the training proceed takes so long for my computer ,so I run training part  using the terminal instead of jupyter notebook,and it works.
To make traing successful,there are two codes Dataset.py and train.py need to be updated,then just a simple terminal command will get training started,should I upload them to the branch TF2.1?

Reply:
@samwang141224 -- No need to train the code.  Just confirm that you 
can generate the model.  And it sounds like you have.